src/Termonad/XML.hs

```diff
@@ -253,3 +257,255 @@ closeTabDoc =
 
 closeTabText :: Text
 closeTabText = toStrict $ renderText def closeTabDoc
+
+preferencesDoc :: Document
+preferencesDoc =
+  [xmlRaw|
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated with glade 3.22.1 -->
```

Comment:
Did you create this in Glade?  Do you have experience with GTK programming, or using Glade?

The other interface XML files from this `Termonad.XML` module are so simple I wrote them by hand, but I'd be interested in moving to Glade for some things if you recommend it.

In particular, I think writing this preferences viewer is a good choice to be done in Glade.

Maybe we should move this block of XML code out to an external file that would be easier to modify with Glade?

Reply:
Thanks for reviewing this. 

Not much to say about c0b6bab. It's definitely more readable than what I wrote and to be honest I wasn't very happy about the way I handled all these Maybes. I was pretty sure that there had to be an easy and nice way to handle them, but I just couldn't figure it out. 

That's why for a beginner like me, this kind of review is very valuable. So thanks again for that.

Yes I created the file with Glade. I don't have experience with Gtk and Glade but I've plenty with Qt and its Designer. And for my experience, it's definitely the way to go when you begin to have a few widgets to layout. You need to have a fast visual feedback when designing the UI, and these tools give you just that. Putting the XML in a separate file, would just accelerate the integration of the UI. So I think this is a good idea.